app/src/js/components/DropDown/dropdown.js

```diff
@@ -111,7 +111,7 @@ const Dropdown = ({
   }, [dispatch, getOptions]);
 
   return (
-    <div className={`filter__item form-group__element filter-${paramKey}`}>
+    <div className={`filter__item form-group__element filter-${paramKey.includes('.') ? paramKey.split('.')[0] : paramKey}`}>
```

Comment:
This is because we might have child properties as the paramKey. In this case it would be `error.Error`, which we wouldn't want as a class name.